genpass/database.py

```diff
@@ -0,0 +1,35 @@
+import sqlite3
+
+
+class DatabaseConnection():
+	def __init__(self):
+		# Connecting to sqlite3 database
```

Comment:
This is the information about method. So it should be doc-string which explains what does this function do.
This suggestion applies to all methods inside class

Reply:
Done!